tests/outcomes/plot/test_pandas/test_drawings_data/test.py

```diff
@@ -0,0 +1,260 @@
+import unittest
+
+from hstest.check_result import correct, wrong
+from hstest.dynamic.dynamic_test import dynamic_test
+from hstest.stage import PlottingTest
+from hstest import TestedProgram
+
+
+class TestSeaborn(PlottingTest):
+    @dynamic_test
+    def test(self):
+
+        import matplotlib
+        import pandas as pd
+
+        program = TestedProgram()
+        program.start()
+
+        if len(self.all_figures) != 22:
+            return wrong(f'Expected 22 plots to be plotted using matplotlib library, found {len(self.all_figures)}')
+
+        # area
+
+        area1 = self.all_figures[0]
+        if area1.type != 'area':
+            return wrong(f'Wrong drawing type {area1.type}. Expected area')
+
+        if 'data' not in area1.data or 'x' not in area1.data or 'y' not in area1.data:
+            return wrong(f"Expected 'data', 'x', 'y' key in the data dict of the area drawing")
+
+        if type(area1.data['data']) != pd.DataFrame:
```

Comment:
In matplotlib there's not `pd.DataFrame` type, in pandas and seaborm there is `pd.DataFrame` type. You should unify them since in tests users can draw in different frameworks but tests should check data, not types. Data should be in predictable format. If the drawing type is `scatter` - then data should be the same for matplotlib, seaborn and pandas libraries. If the data type is `bar` than the data should be the same for all supported libraries as well. That's the point of unifying these 3 libraries under the `Drawing` class.

Reply:
Also, add tests where different libraries draw identical data. The `Drawing.data` should be identical then.